docs/guide/Titanium_SDK/Titanium_SDK_How-tos/Platform_API_Deep_Dives/iOS_API_Deep_Dives/iOS_Background_Services.md

```diff
@@ -432,46 +432,6 @@ Ti.App.iOS.addEventListener('silentpush', function(e) {
 });
 ```
 
-### Subscribing to push notifications
-
-In order to receive the silent remote notification, the device needs to be subscribed to push notifications. Use the [Ti.Network.registerForPushNotifications method](#!/api/Titanium.Network-method-registerForPushNotifications) to register for push notifications and setup callbacks when the application first starts, then use either the Cloud module's `PushNotifications.subscribeToken` or `PushNotifications.subscribe` method to subscribe to a push channel.
-
-The `subscribe` method requires the user to be logged in to Arrow to subscribe and receive push notifications, while `subscribeToken` only relies on the device token.
-
-```javascript
-// Require in the Cloud module
-var Cloud = require('ti.cloud');
-
-// Register for push notification
-Ti.Network.registerForPushNotifications({
-    // Only need to listen to alerts
-    types: [Ti.Network.NOTIFICATION_TYPE_ALERT],
-    success: deviceTokenSuccess,
-    error: deviceTokenError
-});
-
-function deviceTokenSuccess(e) {
-
-    // Subscribes the device to the 'silent_push' channel to listen for silent push alerts
-    // The channel name is arbitrary and can be anything you wish
-    Cloud.PushNotifications.subscribeToken({
-        device_token: e.deviceToken,
-        channel: 'silent_push',
-        type: 'ios'
-    }, function (e) {
-        if (e.success) {
-            Ti.API.info('Subscribed');
-        } else {
-            Ti.API.info('Error:\n' + ((e.error && e.message) || JSON.stringify(e)));
-        }
-    });
-}
-
-function deviceTokenError(e) {
-    Ti.API.info('Failed to register for push notifications: ' + e.error);
-}
-```
-
 ### Sending a silent push notification
```

Comment:
I wonder if it's worth keeping this example as the push pieces are pretty crucial to getting the sample working I think

Reply:
I'll change it so the cloud part is gone and only the `registerForPushNotifications` part with the callbacks :+1: